Videos

ModernTek opened the pull request:
This is initial release of videos part. I have used static images loaded from internet so far to see how it looks like. The next version I am going to use JS to load images, because those videos never will be the same. Please let me know if the solution is acceptable.

Gruximillian commented on src/index.html:
```diff
@@ -224,7 +224,33 @@ <h2>im the main section</h2>
+          <h2 class="developer-videos__name">Mark Abrahamson</h2>
+          <img class="developer-videos__img" src="http://500px.com/graphics/pages/team/squares/oleg.jpg">
+        </figure>
+
```
No need for a blank line here.

Gruximillian commented on src/styles/layout/_developerVideos.scss:
```diff
@@ -0,0 +1,80 @@
+  padding: 0;
+  display: -webkit-flex;
+  display: flex;
+  -webkit-align-items: center;
```
I believe that webpack adds vendor prefixes, so you don't need to bother with those. You can check that if you remove them, and if the still appear in the page inspector.

Gruximillian commented on src/styles/layout/_developerVideos.scss:
```diff
@@ -0,0 +1,80 @@
+  align-content: flex-end;
+ }   
+ .developer-videos__img {
+   border:0;
```
Please add a space after the colon.

Gruximillian commented on src/styles/layout/_developerVideos.scss:
```diff
@@ -0,0 +1,80 @@
+.developer-videos__figure {
+  width: 25em;
+  height: 25em;
+  display:block;
```
Space after the colon.

Gruximillian commented on src/styles/layout/_developerVideos.scss:
```diff
@@ -0,0 +1,80 @@
+  width: 100%;
+}
+.developer-videos-wrapper{
+  max-width: 96rem;
```
Add space between the selector and the opening brace.

Gruximillian commented on src/styles/layout/_developerVideos.scss:
```diff
@@ -0,0 +1,80 @@
+}
+
+.developer-videos__name{
+  color:red;
```
Add spaces before brace and after the colons.

Gruximillian replied:
Almost forgot about this one. Use the `$red` variable and you'll get the correct color for the names.

Gruximillian commented on src/styles/layout/_developerVideos.scss:
```diff
@@ -0,0 +1,80 @@
+  margin-top: 0;
+  margin-bottom: 3em;
+  text-align: center;
+  font-size:  2em;
```
Replace two spaces after the colon with one.

ModernTek commented:
OK. Let me try to do video.

Thanks for comment
> On Jan 20, 2018, at 2:52 PM, Vojislav Grujić <notifications@github.com> wrote:
> 
> @Gruximillian commented on this pull request.
> 
> @ModernTek <https://github.com/moderntek> Natalie, why not use the video element and link a video instead of images? We need videos, and any youtube video can act as a placeholder.
> 
> There are some formatting changes I'd also like you to do, they seem like hair splitting, but it's a good habit to make it very consistent. I'll comment on thos inline.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/OscarRobertRodriguez/developerNews/pull/17#pullrequestreview-90312921>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AQE7s5t0PAG7YeY3Kzjp6_lb8BKPBhe3ks5tMkQKgaJpZM4Rljxr>.
>

Gruximillian commented:
@ModernTek 
Check the spaces on all of your css and try to make then consistent. It's a small thing, but goes a long way in making the code clean.

Gruximillian commented:
Ooops, sorry, I closed it accidentally.

ModernTek commented:
I have added posters and UI custom video controls. I have added two images to index.js. @OscarRobertRodriguez and @Gruximillian please review and let me know Mean time I will write JS for custom video controls

ModernTek commented:
I have added posters and UI for custom video controls. @Gruximillian @OscarRobertRodriguez please review. I will add JS for custom video controls afterwards.

Gruximillian commented:
@ModernTek 
Posters look nice, and the code is good. But the controls need more styling, the alignment, the appearance, colors, all need a bit more work. You could maybe use that `$red` color for the controls to fit it in the site styles, or mabye a bit darker, you can use sass function `darken($red, 25%)` to get a some percentage darker color.

ModernTek commented:
Vojislav,

I will work more on styling. First I would like to finish with JS.

Thank you for comments
> On Jan 26, 2018, at 2:02 PM, Vojislav Grujić <notifications@github.com> wrote:
> 
> @ModernTek <https://github.com/moderntek>
> Posters look nice, and the code is good. But the controls need more styling, the alignment, the appearance, colors, all need a bit more work. You could maybe use that $red color for the controls to fit it in the site styles, or mabye a bit darker, you can use sass function darken($red, 25%) to get a some percentage darker color.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/OscarRobertRodriguez/developerNews/pull/17#issuecomment-360874408>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AQE7s1fDOV9kubFFffZqwYE9Jnl2uAZSks5tOiFPgaJpZM4Rljxr>.
>
